docs/node/installation.md

```diff
@@ -138,7 +145,9 @@ sudo yum install \
     mosquitto-devel \
     nanomsg-devel \
     spdlog-devel \
-    zeromq-devel
+    zeromq-devel \
+    libglib2.0-0 libglib2.0-dev \
```

Comment:
```suggestion
    glib2-devel \
```

Those packages dont exit either..

E.g. on RPM systems, the library is called differently `glib-devel`:
https://fedora.pkgs.org/41/fedora-x86_64/glib2-devel-2.82.1-1.fc41.i686.rpm.html